content/docs/user-guide/gems/reference/ros2/ros2-gem-user-guide.md

```diff
@@ -0,0 +1,209 @@
+| `DriveModel / Axles `                                | List of axles of the vehicle.                                      |
+| `DriveModel / Axles / Axle Wheels `                  | List of wheels in axis.                                            |
+| `DriveModel / Axles / Is it a steering`              | It is ignored in this model.                                       |
+| `DriveModel / Axles / Is it a drive`                 | If it is enabled, the Skid Steering Drive Model will apply velocities to the axis' wheels.|
+| `DriveModel / Axles / Track`                         | Distance between front and rear axis.                              |
+| `DriveModel / Axles / Wheelbase`                     | Distance between left and right wheel.                             |
+| `DriveModel / Vehicles Limits / Linear speed limit ` | Maximum achievable linear speed in meters per second.              |
+| `DriveModel / Vehicles Limits / Angular speed limit` | Maximum achievable angular speed in radians per second.            |
+ #### Manual control
+
+The `VehicleModel` will handle input events with names "steering" and "accelerate". This means you can add an [InputComponent](https://www.o3de.org/docs/user-guide/components/reference/gameplay/input/) to the same entity and define an input map for your input devices (such as keyboard or a game pad) to control the vehicle manually.
+
+You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/).
+
+It is possible to implement your own control mechanisms with this Component.
+
+### Spawner
+
+`ROS2SpawnerComponent` handles spawning entities during simulation. Available spawnables have to be set up as the component's field before the simulation. User is able to define named spawn points inside the Editor. This can be done by adding `ROS2SpawnPointComponent` to a child entity of an entity with `ROS2SpawnerComponent`. During the simulation user can access names of available spawnables and request spawning using ros2 services. The names of services are `/get_available_spawnable_names` and `/spawn_entity` respectivly. GetWorldProperties.srv and SpawnEntity.srv types are used to handle these features. In order to request defined spawn points names user can use `/get_spawn_points_names` service with GetWorldProperties.srv type. Detailed information about specific spawn point (e.g. pose) can be accessed using `/get_spawn_point_info` service with GetModelState.srv type. All used services types are defined in gazebo_msgs package.
+
+- Spawning: spawnable name should be passed in request.name and the position of entity in request.initial_pose
+  - example call: `ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'robot', initial_pose: {position:{ x: 4, y: 4, z: 0.2}, orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}}}'`
```

Comment:
Formatting suggestions: 
	- Bold each type's name
	- Change the example call to use a code block rather than in-line code style
	- Use in-line code style for file names (as well as paths and directories) and code or configuration elements
	- Write in complete sentences and end them with a period

This suggestion is for all items in the list, however I demonstrate it in the first element as such: 
````suggestion
- **Spawning**: The spawnable name must be passed in `request.name` and the position of entity in `request.initial_pose`.
  - Example call: 
  ```
  ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'robot', initial_pose: {position:{ x: 4, y: 4, z: 0.2}, orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}}}
  ```
````